rts/at/tx/pom.xml

```diff
@@ -38,9 +38,18 @@
     <ssl.args>-Drts.usessl=true -Djavax.net.ssl.trustStore=keystore.jks -Djavax.net.ssl.trustStorePassword=123456</ssl.args>
     <!-- jakarta.servlet-api version 6.0.0 does not include jakarta/servlet/SingleThreadModel needed for undertow-->
     <version.jakarta.servlet.jakarta-servlet-api>5.0.0</version.jakarta.servlet.jakarta-servlet-api>
-
   </properties>
-
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-bom</artifactId>
```

Comment:
Even if I usually like the use of bom (i.e in integration tests and quickstarts) I would prefer not to have the quarkus-bom in our lra api module. Let's see what @mmusgrov think about it.

Reply:
I think now I have used only one dependency i.e quarkus-resteasy. SO there is no sense of using quarkus-bom.